models/buffer.go

```diff
@@ -49,7 +49,7 @@ type BufferStats struct {
 }
 
 // NewBuffer returns a new empty Buffer with the given capacity.
-func NewBuffer(name string, alias string, capacity int, strategy string, path string) (Buffer, error) {
+func NewBuffer(id, name, alias string, capacity int, strategy, path string) (Buffer, error) {
```

Comment:
Do you need the `alias` still? Maybe `name, id` would be more intuitive?

Reply:
The alias is still used for the buffer stats for both buffer strategies, though it isn't used for either buffer configuration otherwise. I can swap ID and name here though